x/nexus/keeper/keeper.go

```diff
@@ -159,6 +159,23 @@ func (k Keeper) EnqueueForTransfer(ctx sdk.Context, sender exported.CrossChainAd
 	k.Logger(ctx).Info(fmt.Sprintf("Transfer of %s to cross chain address %s in %s successfully prepared",
 		asset.Amount.String(), recipient.Address, recipient.Chain.Name))
 
+	// merging all transfers for the specified recipient
```

Comment:
This is more related to the fee calculation above. Now that we are merging transfers, fee calculation should not be done per individual transfer but whenever mint/withdrawal is executed for the whole. Please add a TODO and create an issue.

Reply:
added a TODO comment, gonna open an issue